### ISSUE TITLE
[Android] Update publisher/creators semi-verified/connected state text: "This Brave Verified Creator has not yet configured their account to receive contributions from Brave Users. Any tips you send will remain in your wallet until they complete this process." 

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#15759]
## Description

Android version of https://github.com/brave/brave-browser/issues/14330.

**Comment by deeppandya** [comment]
![Screenshot_20210603-175228](https://user-images.githubusercontent.com/5146988/120716834-904b7400-c494-11eb-95fe-53c709382396.png)

**Comment by srirambv** [comment]
Blocked on #16570

**Comment by srirambv** [comment]
Marking as QA/No as this is superseded by #16570 which is not going to be part of 1.27.x so this will still be an issue on 1.27.x
